server/planning/assignments/assignments.py

```diff
@@ -336,6 +338,53 @@ def send_assignment_notification(self, updates, original=None, force=False):
         else:
             event_item = None
 
+        # Create the ICS file to be added to the email usable in google calendar.
+        ical = Calendar()
+        ical.add('BEGIN', 'VCALENDAR')
+        ical.add('PRODID', '-//Superdesk//NTB//EN')
+        ical.add('VERSION', '2.0')
+        ical.add('BEGIN', 'VEVENT')
+        ical.add('CLASS', 'PUBLIC')
+
+        if assignment['planning']['scheduled']:
+            scheduled_time = assignment['planning']['scheduled']
+            ical.add('DTSTART', scheduled_time)
+            ical.add('DTEND', scheduled_time)
+
+        UID = str(assignment['_id'])
+        url = client_url + '#/workspace/assignments?assignment=' + UID
+        summary = get_assginment_name(assignment)
+        priority = assignment['priority']
+        created = assignment['_created']
+        updated = assignment['_updated']
+
+        ical.add('UID', UID)
+        ical.add('SUMMARY;LANGUAGE=EN-US', summary)
```

Comment:
This entry should be using the default language as defined in `app.config` (also used in the `PRODID`)